samples: bluetooth: update minimum key size in security parameters (#845)
nrfconnect/sdk-nrf-bm: doc/nrf-bm/release_notes/release_notes_changelog.rst

Bluetooth LE samples
--------------------

* Updated:

>    * All samples with :ref:`lib_peer_manager` to use a minimum key size of 16 bytes in the security parameter default configuration.

Reviewer: ```suggestion
   * All samples that use the :ref:`lib_peer_manager` library to use a minimum encryption key size of 16 bytes in their default security parameters.
     Peers that do not support 16-byte encryption keys can no longer pair with these samples.
     The following samples are affected:
      * :ref:`ble_bms_sample`
      * :ref:`ble_cgms_sample`
      * :ref:`ble_hids_keyboard_sample`
      * :ref:`ble_hids_mouse_sample`
      * :ref:`ble_hrs_sample`
      * :ref:`ble_hrs_central_sample`
      * :ref:`peripheral_nfc_pairing_sample`
```

Author: Thanks. Updated.

>    * The following samples and applications that do not support pairing to call the :c:func:`sd_ble_gatts_sys_attr_set` function only in response to the :c:macro:`BLE_GATTS_EVT_SYS_ATTR_MISSING` event and not as a response to a :c:macro:`BLE_GAP_EVT_CONNECTED` event:

Reviewer: ```suggestion
   * The following samples and applications, which do not support pairing, to call the :c:func:`sd_ble_gatts_sys_attr_set` function only in response to the :c:macro:`BLE_GATTS_EVT_SYS_ATTR_MISSING` event, and not in response to the :c:macro:`BLE_GAP_EVT_CONNECTED` event:
```

Author: Updated.
